### PR TITLE
feature/hip-12-add-GitHub-ci

### PR DESCRIPTION
**Pull request by yribeiro**

### CODE REVIEW

File: .github/workflows/run_docs.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        installer-parallel: true
+
+    #----------------------------------------------
+    # install dependencies
+    #----------------------------------------------
+    - name: Install dependencies
+      run: poetry install --no-interaction
+
+    #----------------------------------------------
+    # Runs precommit using the repo config file
```
Comment (mateimol) [inline .github/workflows/run_docs.yml]: shouldn't this line say that it checks or builds docs? instead of saying it runs the pre-commit ?

**Review by mateimol — CHANGES_REQUESTED** [review]
Change comment line (line 45) in run_docs.yml to match the actual file actions.

File: .github/workflows/run_docs.yml (hunk 1 above)
Comment (yribeiro) [reply .github/workflows/run_docs.yml]: yup, you're right. Updated.

**Review by mateimol — APPROVED** [review]
All good.

**Pull request merged**